content/en/serverless/aws_lambda/configuration.md

```diff
@@ -506,9 +551,64 @@ To filter traces before sending them to Datadog, see [Ignoring Unwanted Resource
 
 To scrub trace attributes for data security, see [Configure the Datadog Agent or Tracer for Data Security][23].
 
-## Disable trace collection
+## Enable/disable trace collection
+
+Trace collection through the Datadog Lambda extension is enabled by default. 
+
+If you want to stop collecting traces from your Lambda functions, follow the instructions below:
```

Comment:
```suggestion
If you want to start collecting traces from your Lambda functions, apply the configurations below:
```

Reply:
It looks like these commands and configurations are for enabling traces rather than stopping them, but let me know if I'm missing anything. Also suggested to update the wording, since there aren't any actual instructions being supplied.